src/VectorizedSolver.cpp

```diff
@@ -322,76 +328,52 @@ void VectorizedSolver::normalizeFluxes() {
 void VectorizedSolver::computeFSRSources() {
 
   int tid;
-  FP_PRECISION scatter_source;
-  FP_PRECISION fission_source;
-  FP_PRECISION* nu_sigma_f;
-  FP_PRECISION* sigma_s;
-  FP_PRECISION* sigma_t;
-  FP_PRECISION* chi;
   Material* material;
+  FP_PRECISION* sigma_t;
+  FP_PRECISION* sigma_s;
+  FP_PRECISION* fiss_mat;
+  FP_PRECISION scatter_source, fission_source;
 
-  int size = _num_FSRs * _num_groups * sizeof(FP_PRECISION);
+  int size = _num_threads * _num_groups * sizeof(FP_PRECISION);
```

Comment:
The reason is that the ``fission_sources`` and ``scatter_sources`` arrays are "scratchpad" arrays for each thread. The outer loop is over FSRs while the inner loops are such that each thread accumulates the source *into* each group *from* every group. 

Now one of the things that I realized when I was revisiting this code to make this change is that this code block will suffer from the same false sharing issue as was once an issue for the transport sweep. Since computing FSR sources takes up a relatively small fraction of time compared to the sweeps, I decided to keep this structure in place since it makes the code (slightly) more readable at the expense of poor parallel scaling. I'm happy to change this if you think it necessary.

Perhaps you have a better approach in mind. If so, I can put it into place or you can submit a separate PR.